src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
+html {
+  height: 100%;
+}
+
 body {
   margin: 0;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__minutes {
+    position: absolute;
+    left: calc(50% - 1.5vmin);
+    top: calc(50% - 20vmin);
+    display: block;
+    width: 3vmin;
+    height: 20vmin;
+    background-color: #0700ff;
+    animation: nev 3600s infinite steps(60);
```

Comment:
The animation property here is set to `nev 3600s infinite steps(60);`. It seems like you're trying to create a minute hand for a clock. However, the `steps()` function might not be the best choice here. The `steps()` function creates a stepped animation, where the animation jumps from one keyframe to the next instantly, without any transition in between. For a clock hand, a smoother transition might be more appropriate. Consider using a different timing function, such as `linear`.

Reply:
this is according to the task